test/test_nonhydrostatic_models.jl

```diff
@@ -102,10 +102,14 @@ using Oceananigans.Grids: required_halo_size_x, required_halo_size_y, required_h
         @info "  Testing model construction with single tracer and nothing tracer..."
         for arch in archs
             for grid in grids
-                model = NonhydrostaticModel(grid; tracers=:c, buoyancy=nothing)
+                model = NonhydrostaticModel(grid;
+                                            tracers = :c,
+                                            buoyancy = nothing)
                 @test model isa NonhydrostaticModel
 
-                model = NonhydrostaticModel(grid; tracers=nothing, buoyancy=nothing)
+                model = NonhydrostaticModel(grid;
+                                            tracers = nothing,
+                                            buoyancy = nothing)
```

Comment:
```suggestion
                model = NonhydrostaticModel(grid)
```

those are default so not need to set directly... might be a lot of that which can all be cleaned up